### PR TITLE
add sely's notes

### PR DESCRIPTION
**Pull request by ssun098**
lul

### CODE REVIEW

**Comment by ssun098** [comment]
@MaxWofford i can't assign anyone

@zachlatta could you give me contributor access?

File: meetings/15-08-27_selys_ongoing_notes [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Sely's Notes
```
Comment (MaxWofford) [inline meetings/15-08-27_selys_ongoing_notes]: Headings should have newlines after them.

File: meetings/15-08-27_selys_ongoing_notes [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+## Meeting with Mr. Smith
+### 08/27/2015, 11:50am
+Purpose was to ask about his interest in being an adviser for our club.
+
+Dan said "Well, what do you want to know?" --> bad start, seems like an
```
Comment (MaxWofford) [inline meetings/15-08-27_selys_ongoing_notes]: Who is Dan?

**Comment by MaxWofford** [comment]
@ssun098 Here's a good example of how we're trying to organize our notes: https://github.com/hackedu/meta/blob/master/meetings/15-08-24_planning_session.org

**Comment by zachlatta** [comment]
Does it make sense to include these in our main hackEDU repo? It seems out
of scope.

On Thu, Aug 27, 2015 at 4:18 PM, Max Wofford <notifications@github.com>
wrote:

> @ssun098 <https://github.com/ssun098> Here's a good example of how we're
> trying to organize our notes:
> https://github.com/hackedu/meta/blob/master/meetings/15-08-24_planning_session.org
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackedu/meta/pull/116#issuecomment-135580897>.
>



-- 
Zach Latta
hackEDU | zach@hackedu.us

**Comment by ssun098** [comment]
@zachlatta yeah I was concerned about that

**Comment by zachlatta** [comment]
Going to hold off on merging for now because I think it's out of scope. Let me know if anyone has any objections!

**Pull request closed without merging**